lib/routes/docschina/maintainer.ts

```diff
@@ -1,3 +1,3 @@
 export default {
-    '/jsweekly': ['daijinru'],
+    '/jsweekly': ['daijinru', 'hestudy'],
```

Comment:
https://github.com/DIYgod/RSSHub/blob/59fcfdb9251f8c8f6cd7228b22a7786eb962bbf1/website/docs/routes/programming.mdx#L1144
```suggestion
    '/weekly/:category?': ['daijinru', 'hestudy'],
```